### PR TITLE
[WFLY-14206] Integrate MP Metrics 3.0 into EAP

### PR DESCRIPTION
**Pull request by jasondlee**

### CODE REVIEW

**Review by jmesnil — APPROVED** [review]
That looks good to me

**Review by maeste — APPROVED** [review]

File: microprofile/WFLY-14206-Integrate-MP-Metrics-3.0-into-EAP.adoc [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+=== Related Issues
+
+* https://issues.redhat.com/browse/EAP7-1592
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
```
Comment (jstourac) [inline microprofile/WFLY-14206-Integrate-MP-Metrics-3.0-into-EAP.adoc:31]: `mailto:jstourac@redhat.com[Jan Stourac]`

**Comment by jstourac** [comment]
Hi @jasondlee, LGTM, could you please just add a me as a QE contact? Thanks.

File: microprofile/WFLY-14206-Integrate-MP-Metrics-3.0-into-EAP.adoc (hunk 1 above)
Comment (jasondlee) [reply microprofile/WFLY-14206-Integrate-MP-Metrics-3.0-into-EAP.adoc:31]: Done. Thanks!

**Review by jstourac — APPROVED** [review]

**Pull request merged**